### ISSUE TITLE
NullPointerException: Cannot read field "x" because "mousePos" is null

### ISSUE THREAD

**Issue by Tijs-B** [ubipo/shrinkwrap#3]
Hi, I'm having troubles using the shrinkwrap plugin on my Mac computer with the latest version of JOSM. When I try to use the balloon function of shrinkwrap by using the keyboard shortcut, JOSM immediately reports a crash of the plugin. 

## Reproducing
1. Install shrinkwrap (no other plugins used) and restart JOSM
2. Create an enclosed area
3. Use the keyboard shortcut (Option-Shift-B in my case) while the mouse is inside the area to balloon the area

## What happens
JOSM reports a crash of the shrinkwrap plugin

## What should happen
The area should be ballooned (a way should be created of the ballooned area)

## Full crash data
```
{{{
Revision:17919
Build-Date:2021-06-02 20:30:01

Identification: JOSM/1.5 (17919 en) Mac OS X 10.14.6
OS Build number: Mac OS X 10.14.6 (18G9028)
Memory Usage: 770 MB / 2048 MB (155 MB allocated, but free)
Java version: 16.0.1+9, Azul Systems, Inc., OpenJDK 64-Bit Server VM
Look and Feel: com.apple.laf.AquaLookAndFeel
Screen: Display 69733632 1440×900 (scaling 2.00×2.00) Display 1127244958 3008×1692 (scaling 2.00×2.00)
Maximum Screen Size: 3008×1692
Best cursor sizes: 16×16→16×16, 32×32→32×32
System property file.encoding: UTF-8
System property sun.jnu.encoding: UTF-8
Locale info: en_BE
Numbers with default locale: 1234567890 -> 1234567890
VM arguments: [-Djpackage.app-version=17919, --add-exports=java.desktop/com.apple.eawt=ALL-UNNAMED, --add-exports=java.desktop/com.sun.imageio.plugins.jpeg=ALL-UNNAMED, --add-exports=java.desktop/com.sun.imageio.spi=ALL-UNNAMED, --add-opens=java.base/java.lang=ALL-UNNAMED, --add-opens=java.base/java.nio=ALL-UNNAMED, --add-opens=java.base/jdk.internal.loader=ALL-UNNAMED, --add-opens=java.base/jdk.internal.ref=ALL-UNNAMED, --add-opens=java.desktop/javax.imageio.spi=ALL-UNNAMED, --add-opens=java.desktop/javax.swing.text.html=ALL-UNNAMED, --add-opens=java.prefs/java.util.prefs=ALL-UNNAMED, -Djpackage.app-path=/Applications/JOSM.app/Contents/MacOS/JOSM]
Dataset consistency test: No problems found

Plugins:
+ shrinkwrap (v1.0.4)

Map paint styles:
- https://josm.openstreetmap.de/josmfile?page=Styles/Maxspeed&zip=1
- ${HOME}/Downloads/josm_style_grb.mapcss
- https://josm.openstreetmap.de/josmfile?page=Styles/Lane_and_Road_Attributes&zip=1

Last errors/warnings:
- 00007.503 W: Failed to load Mappaint styles from '${HOME}/Downloads/josm_style_grb.mapcss'. Exception was: java.nio.file.NoSuchFileException: ${HOME}/Downloads/josm_style_grb.mapcss
- 00007.503 E: java.nio.file.NoSuchFileException: ${HOME}/Downloads/josm_style_grb.mapcss
- 00087.368 E: Handled by bug report queue: java.lang.NullPointerException: Cannot read field "x" because "mousePos" is null



=== REPORTED CRASH DATA ===
BugReportExceptionHandler#handleException:
No data collected.

Warning issued by: BugReportExceptionHandler#handleException

=== STACK TRACE ===
Thread: AWT-EventQueue-0 (23) of main
java.lang.NullPointerException: Cannot read field "x" because "mousePos" is null
	at net.pfiers.shrinkwrap.BalloonAction.actionPerformed(BalloonAction.kt:55)
	at java.desktop/javax.swing.SwingUtilities.notifyAction(Unknown Source)
	at java.desktop/javax.swing.JComponent.processKeyBinding(Unknown Source)
	at java.desktop/javax.swing.KeyboardManager.fireBinding(Unknown Source)
	at java.desktop/javax.swing.KeyboardManager.fireKeyboardAction(Unknown Source)
	at java.desktop/javax.swing.JComponent.processKeyBindingsForAllComponents(Unknown Source)
	at java.desktop/javax.swing.JComponent.processKeyBindings(Unknown Source)
	at java.desktop/javax.swing.JComponent.processKeyEvent(Unknown Source)
	at java.desktop/java.awt.Component.processEvent(Unknown Source)
	at java.desktop/java.awt.Container.processEvent(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(Unknown Source)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(Unknown Source)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(Unknown Source)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(Unknown Source)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.Component.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$5.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Unknown Source)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
}}}
```

**Comment by ubipo** [comment]
Couldn't reproduce. I did add a warning (5599499) when this occurs so it at least doesn't lead to a crash.

**Comment by ubipo** [comment]
[Published as v1.0.6](https://github.com/ubipo/shrinkwrap/releases/tag/v1.0.6) and available in the plugin browser (update JOSM to 19017 or later)

**Comment by Tijs-B** [comment]
I haven't used this plugin in ages but will let you know if I encounter it again. Thanks!
